pkg/apis/security/v1alpha1/types.go

```diff
@@ -185,6 +185,9 @@ const (
 	RuleActionAllow RuleAction = "Allow"
 	// RuleActionDrop describes that rule matching traffic must be dropped.
 	RuleActionDrop RuleAction = "Drop"
+	// RuleActionReject describes that rule matching traffic must be rejected and the
```

Comment:
I know you are following the previous comments, but they are not easy to understand.

How about: RuleActionReject indicates that the traffic matching the rule must be rejected...

Reply:
Got you. Should I change previous comments as well?

Reply:
Yes, we should make all consistent.

Reply:
Done.